source/environment-variables.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Environment Variables
+description: List of environment variables that you can use with your Meteor application.
+---
+
+Here's a list of the environment variables you can provide to your application. All of these work both in development and production mode.
```

Comment:
Actually, not all of these work in development.

`METEOR_SETTINGS` definitely doesn't work in development and I'm not entirely positive about `BIND_IP` and `PORT` (this is typically done with `--port 0.0.0.0:6000`).  Also, `ROOT_URL` _might_ be done with `--server https://example.com/path/` in development.

I'd need to test to refresh my memory exactly as I don't do them very often – it may also be in the docs somewhere else.

We should change this sentence and then put a note next to each variable about where it works, similar to the `client` / `server` / `everywhere` notation in the rest of the docs.

Will try to revisit this next week if you don't get to it before me!